samples/other/file-transfer-s3-to-s3/src/main/java/org/eclipse/dataspaceconnector/transfer/demo/api/ClientApiController.java

```diff
@@ -115,9 +88,13 @@ private DataRequest createRequest(
             String destinationRegion
     ) {
 
+        if (1 == 1) {
```

Comment:
If this always throws an exception, why not deleting the whole module?

Reply:
oops, I put this in during debugging and never deleted it :/